README.md

```diff
@@ -1,5 +1,5 @@
 # AWS SMTP Relay
-> SMTP server to relay emails via AWS SES API using IAM roles.
+> SMTP server to relay emails via AWS SES or Pinpoint API using IAM roles.
```

Comment:
Maybe this sentence reads better the following way:
> SMTP server to relay emails via Amazon SES or Amazon Pinpoint using IAM roles.

Reply:
Done.